src/bench/bench_bitcoin.cpp

```diff
@@ -29,6 +29,7 @@ static void SetupBenchArgs(ArgsManager& argsman)
     argsman.AddArg("-min_time=<milliseconds>", strprintf("Minimum runtime per benchmark, in milliseconds (default: %d)", DEFAULT_MIN_TIME_MS), ArgsManager::ALLOW_ANY | ArgsManager::DISALLOW_NEGATION, OptionsCategory::OPTIONS);
     argsman.AddArg("-output_csv=<output.csv>", "Generate CSV file with the most important benchmark results", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     argsman.AddArg("-output_json=<output.json>", "Generate JSON file with all benchmark results", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-one-iteration", "Run benchmarks for only one iteration (for testing purposes)", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
nit, sort here and below

Reply:
The other args are snakecased and this is kebabcase; maybe good to use one style or the other.

Reply:
Yes, I agree. I'll convert all the arguments to kebab-case (which is customary for UNIX-style arguments) in another commit.